src/renderer/services/utils/file-utils.ts

```diff
@@ -0,0 +1,16 @@
+import fs from "fs";
+
+export interface ICopyFileStatus {
+    success: boolean;
+    message: string
+}
+
+export function copyFile(fromPath: string, toPath: string, readOptions = "utf-8"): ICopyFileStatus {
+    try {
+        const fileContent = fs.readFileSync(fromPath, readOptions);
+        fs.writeFileSync(toPath, fileContent);
```

Comment:
We ar enot int 2003, use pipes or native copy:
https://stackoverflow.com/questions/11293857/fastest-way-to-copy-file-in-node-js

:smile: